salt/modules/cp.py

```diff
@@ -392,6 +402,7 @@ def get_url(path, dest="", saltenv="base", makedirs=False, source_hash=None):
         salt '*' cp.get_url salt://my/file /tmp/this_file_is_mine
         salt '*' cp.get_url http://www.slashdot.org /tmp/index.html
     """
+    # TODO
```

Comment:
What is the TODO for?

Reply:
oh... to remember there is still work to do :) thanks for catching it.

Reply:
I added the same query to retrieve saltenv from minion config. I hope it's compatible with the `get_url` intended behaviour.